src/components/pages/Admin/components/RegistrationRequests/registrationTableColumns.tsx

```diff
@@ -18,62 +18,181 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
-import { IconButton, StatusTag, Chip } from '@catena-x/portal-shared-components'
+import { IconButton, Typography } from '@catena-x/portal-shared-components'
 import type { GridColDef } from '@mui/x-data-grid'
 import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
-import ArticleOutlinedIcon from '@mui/icons-material/ArticleOutlined'
 import dayjs from 'dayjs'
-import uniqueId from 'lodash/uniqueId'
-import type {
-  ApplicationRequest,
-  ProgressButtonsType,
+import {
+  ApplicationRequestStatus,
+  ProgressStatus,
+  type ApplicationRequest,
+  type ProgressButtonsType,
 } from 'features/admin/applicationRequestApiSlice'
 import EditIcon from '@mui/icons-material/Edit'
-import './RegistrationRequests.scss'
-import CheckList from './components/CheckList'
 import type i18next from 'i18next'
+import { Progress } from 'components/shared/basic/Progress'
+import './RegistrationRequests.scss'
 
 // Columns definitions of Registration Request page Data Grid
 export const RegistrationRequestsTableColumns = (
   t: typeof i18next.t,
-  handleDownloadDocument: (
-    appId: string,
-    documentId: string,
-    documentType: string
-  ) => void,
   showConfirmOverlay?: (applicationId: string) => void,
-  onConfirmationCancel?: (applicationId: string, name: string) => void,
-  onChipButtonSelect?: (button: ProgressButtonsType, id: string) => void
+  onConfirmationCancel?: (applicationId: string, name: string) => void
 ): Array<GridColDef> => {
-  return [
-    {
-      field: 'dateCreated',
-      headerName: t('content.admin.registration-requests.columns.date'),
-      flex: 1.5,
-      disableColumnMenu: true,
-      valueGetter: ({ row }: { row: ApplicationRequest }) =>
-        dayjs(row.dateCreated).format('YYYY-MM-DD'),
-    },
+  const getStatusProgress = (row: ApplicationRequest) => {
+    const todoItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.TO_DO
+    ).length
+    const inprogressItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.IN_PROGRESS
+    ).length
+    const doneItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.DONE
+    ).length
+    const failedItems = row.applicationChecklist.filter(
+      (checklist: ProgressButtonsType) =>
+        checklist.statusId === ProgressStatus.FAILED
+    ).length
+
+    let items = {
+      TO_DO: todoItems,
+      IN_PROGRESS: inprogressItems,
+      DONE: doneItems,
+      FAILED: failedItems,
+    }
```

Comment:
haven't checked in detail but this looks like it can all (line 42-65) be replaced by a single call to a groupBy function like: https://www.geeksforgeeks.org/lodash-_-groupby-method/